projects/storefrontlib/src/cms-components/myaccount/order/cancellations-returns/return-order/return-order.component.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { Component, Input, Output, EventEmitter, Type } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { of } from 'rxjs';
+import {
+  Order,
+  OrderEntry,
+  CancellationReturnRequestEntryInput,
+  RoutingService,
+} from '@spartacus/core';
+import { OrderDetailsService } from '../../order-details/order-details.service';
+import { ReturnOrderComponent } from './return-order.component';
+
+@Component({
+  template: '',
+  selector: 'cx-cancellation-return-items',
+})
+class MockCancellationReturnItemsComponent {
+  @Input() entries: OrderEntry[];
+  @Input() confirmation = false;
+  @Input() cancelOrder = true;
+  @Output() confirm = new EventEmitter<CancellationReturnRequestEntryInput[]>();
+}
+
+class MockRoutingService {
+  go = jasmine.createSpy('go');
+}
+
+const mockOrder: Order = {
+  code: '1',
+  entries: [
+    { entryNumber: 0, returnableQuantity: 1 },
+    { entryNumber: 1, returnableQuantity: 0 },
+    { entryNumber: 3, returnableQuantity: 5 },
+  ],
+  created: new Date('2019-02-11T13:02:58+0000'),
+  returnable: true,
+};
+
+class MockOrderDetailsService {
+  _cancellationReturnRequestInputs: CancellationReturnRequestEntryInput[];
+
+  getOrderDetails() {
+    return of(mockOrder);
+  }
+
+  get CancellationReturnRequestInputs(): CancellationReturnRequestEntryInput[] {
+    return this._cancellationReturnRequestInputs;
+  }
+
+  set CancellationReturnRequestInputs(
+    values: CancellationReturnRequestEntryInput[]
+  ) {
+    this._cancellationReturnRequestInputs = values;
+  }
+}
+
+describe('OrderDetailItemsComponent', () => {
```

Comment:
The component name should be `ReturnOrderComponent`

Reply:
👍